usagov/vote-gov-nvrf-app
# Feature/vote 2901/fetch data vote gov

Ticket: https://cm-jira.usa.gov/browse/VOTE-2901

Description: Fetch data from the vote.gov url while running the app in all environments (local, dev, stage, prod)

Testing steps:
1. Open the form locally and confirm data is loading.
2. Field out all fields and confirm the PDF is generated.
